utils/R/get.analysis.filenames.r

```diff
@@ -23,7 +23,7 @@ ensemble.filename <- function(settings, prefix = "ensemble.samples", suffix = "R
     ensemble.id <- "NOENSEMBLEID"
   }
   
-  ensemble.dir <- settings$outdir
+  ensemble.dir <- file.path(settings$outdir, 'ensemble.samples')
```

Comment:
This appears to put the ensemble samples in their own directory rather than in the base outdir. This will make them unaccessible via the web interface. Not sure the reason for this change